src/kube-runtime/src/plugins/common/post.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+# Post-script template
+# Generated script here
```

Comment:
what does `common/*.sh` for? shouldn't you put `common/` under `cmd/`?

Reply:
Because multiple plugins may share same script template, we just need one common template if plugin was not use its own.

Reply:
Removed templates. Use /bin/sh to call